lib/features/auth/presentation/providers/register/register_notifier.dart

```diff
@@ -18,13 +21,24 @@ class RegisterNotifier extends Notifier<RegisterState> {
 
   // Implement your register logic here
   void register() async {
-
     state = state.updateLoading(true);
 
-
-
     // Perform register request...
-
+    try {
+      await authApi.register({
+        'name': state.name,
+        'email': state.email,
+        'password': state.password,
+        'password_confirmation': state.passwordConfirmation,
+      });
+      appRouter.replaceAll([const MainRoute()]);
```

Comment:
why replaceAll ?

Reply:
@mhmdajoor5  still don't understand navigation system, but replaceAll to prevent go back to register page, after go to home page